tests/src/test_cases/libs/test_freetype.c

```diff
@@ -511,6 +511,7 @@ void test_freetype_outline_rendering_test(void)
 
     font_italic->release_glyph(font_italic, &g);
 
+    lv_freetype_font_delete(font_italic);
```

Comment:
Should the font in line 405 also be deleted?
`    lv_font_t * font_italic = lv_freetype_font_create("./src/test_files/fonts/noto/NotoSansSC-Regular.ttf",
`

Reply:
I think that's no need🫣
It's still refs by style and label object.